test/verify/machineslib.py

```diff
@@ -1133,13 +1141,18 @@ class TestCreateConfig:
         MAGEIA_VENDOR = 'Mageia'
         MAGEIA_3_FILTERED_OS = 'Mageia 3'
 
+        LIBVIRT_CONNECTION = {
+            'session': 'QEMU/KVM User connection',
+            'system': 'QEMU/KVM System connection'}
```

Comment:
Why this needs to be internationalized? It's tests code, it's always going to be run in English. (I don't know how translate strings in the tests as well)

Also I don't really see a straight way to use an id instead of this string :/ We check for the select button content in ```selectFromDropdown``` function,  and this is a function that is used all around the machines tests. If I have to drop checking the Button value and instead use id I 'll have to adjust all tests that use this function. It's good suggestions but let's not make it part of this PR please!